core/eolearn/tests/test_core_tasks.py

```diff
@@ -99,11 +104,15 @@ def test_deepcopy(patch):
 
 
 def test_partial_copy(patch):
-    partial_copy = DeepCopyTask(features=[(FeatureType.MASK_TIMELESS, "mask"), FeatureType.BBOX]).execute(patch)
+    partial_copy = DeepCopyTask(
+        features=[(FeatureType.MASK_TIMELESS, "mask"), (FeatureType.MASK_TIMELESS, "LULC"), FeatureType.BBOX]
```

Comment:
here you copy both MASK_TIMELESS features and in the next case you copy both SCALAR features...

Reply:
why are there changes if it's not part of this MR?

Reply:
old aka here curnt `test_partial_copy` in test copy FeatureType insted of FeatureSpec Fixtuer EOPatch were updated for `test_merge_features` i did minimal change posible on test copy that test are Passing 

if merge is going to use its owen Fixtuer EOPatch the change is going to be reverted on this branch